packages/apps-config/src/endpoints/util.ts

```diff
@@ -52,9 +54,12 @@ export function expandEndpoint (t: TFunction, { dnslink, genesisHash, homepage,
     .map(([host, value], index): LinkOption => ({
       ...base,
       dnslink: index === 0 ? dnslink : undefined,
+      isLightClient: typeof value === 'string' ? false : value.type === 'substrate-connect' as EndpointType,
```

Comment:
```suggestion
      isLightClient: typeof value === 'string' ? false : value.type === 'substrate-connect' as EndpointType,
```

Not sure why the `as EndpointType` is needed here. Really should not be.